astropy/visualization/scripts/fits2bitmap.py

```diff
@@ -234,14 +250,24 @@ def main(args=None):
     )
     args = parser.parse_args(args)
 
+    if args.min_cut is not None:
+        warnings.warn('The "--min_cut" argument is deprecated. Use "--vmin" instead.')
+        if args.vmin is None:
+            args.vmin = args.min_cut
```

Comment:
Combined with my other suggestion to use mutually exclusive options, the condtional becomes redundant.
```suggestion
        args.vmin = args.min_cut
```

Reply:
done